megameklab/src/megameklab/util/UnitPrintManager.java

```diff
@@ -155,23 +155,23 @@ public static List<PrintRecordSheet> createSheets(List<? extends BTObject> entit
                     }
                 } else if (unit instanceof BattleArmor) {
                     baList.add((BattleArmor) unit);
-                    if (singlePrint || baList.size() > 4) {
+                    if (singlePrint || PrintSmallUnitSheet.fillsSheet(baList, options)) {
```

Comment:
Here too

Reply:
I don't think it makes sense to add parentheses here. There's only one operator in the new expression, although the code that was here before could have used them.